jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/RemoteTaskServiceImpl.java

```diff
@@ -77,6 +80,33 @@ public TaskSummary getTask(String serverTemplateId,
         }
     }
 
+    @Override
+    public TaskSummary getTaskWithSLA(String serverTemplateId, String containerId, Long taskId, Long workItemId) {
```

Comment:
getTask already returns the _workItemId_ it doesnt need to be received as parameter in here.

Reply:
@cristianonicolai I added this new getTaskWithSLA because the getTask didn't fill the workItemId. The idea was to reuse the info (workitem) retrieved by de dataset query (maintained wrongly at view) to be able to refresh the task data and  sla info retrieving the nodeinstance related. Maybe we can simplify this, updating the service find bytaskId to fill this info. does this makes sense?

Reply:
@nmirasch I think so, from what I can see, _workItemId_ is available in the _getTask_ method just not set in the _TaskSummary_ object. I guess we just need to ensure that _getTask_ still work for when the container is not available, with this new info about the node instance.